xrpl/models/transactions/__init__.py

```diff
@@ -16,6 +16,12 @@
 from xrpl.models.transactions.escrow_cancel import EscrowCancel
 from xrpl.models.transactions.escrow_create import EscrowCreate
 from xrpl.models.transactions.escrow_finish import EscrowFinish
+from xrpl.models.transactions.metadata import (
+    CreatedNode,
+    DeletedNode,
+    ModifiedNode,
```

Comment:
IMO these don't need to be exported at the transaction level. You can still import them from `xrpl.models.transactions.metadata`